jep/7/README.adoc

```diff
@@ -132,11 +183,110 @@ Workarounds for this include downloading affected plugins from Artifactory, and
+
+Capitomcat::
+Last released **6 years ago**. +
+Suggestion: Install Ruby and Capistrano and use `sh` or `bat` to invoke them from the command line.
+
+Commit Message Trigger::
+Last released **7 years ago**. +
+Suggestion: Use `sh`, `bat`, or other scripts to read git commit messages and conditionally execute Pipeline steps.
+
+Git notes::
+Last released **9 years ago**. +
+Suggestion: Use `sh`, `bat`, or other scripts to run `git` to annotate commits.
+
+rbenv::
+Last released **5 years ago**. +
+Suggestion: Use `sh` or `bat` to run `rbenv` from the command line.
+
+Chef::
+Last released **6 years ago**. +
+Suggestion: Use `sh` or `bat` to run `chef` from the command line.
+
+CI Skip::
+Last released **7 years ago**. +
+Suggestion: Use the plugin:github-scm-trait-commit-skip[GitHub Commit Skip SCM Behaviour], plugin:bitbucket-scm-trait-commit-skip[Bitbucket Commit Skip SCM Behaviour], or plugin:scmskip[SCM Skip] to skip builds based on the content of commit messages.
+Alternately, use `sh`, `bat`, or other scripts to read git commit messages and conditionally execute Pipeline steps.
+
+InstallShield::
+Last released **7 years ago**. +
+Suggestion: Use `sh`, `bat`, or other scripts to run InstallShield.
+
+MySQL Job Databases::
+Last released **7 years ago**. +
+Suggestion: Use link:https://github.com/jbox-web/job-database-manager-mysql[Jenkins Job Database Manager Plugin for MySQL].
+
+Pathignore::
+Last released **9 years ago**. +
+Suggestion: Use the path ignore features of various plugins or use `sh`, `bat`, or other scripts to read git commit messages and conditionally execute Pipeline steps.
+
+Perl::
+Last released **8 years ago**. +
+Suggestion: Use `sh` or `bat` to run `perl` from the command line.
+
+pry::
+Last released **9 years ago**. +
+Suggestion: Use the Jenkins groovy console and its interface from the Jenkins command line interface.
+
+Single Use Slave::
+Last released **6 years ago**. +
+Suggestion: Use cloud agents (plugin:amazon-ecs[Fargate], plugin:azure-container-agents[Azure Container Instances], plugin:docker-workflow/[Docker], etc.) to allocate agents for a single use and then release them.
+
+Travis YML::
+Last released **4 years ago**. +
+Suggestion: Rewrite the travis.yml file as a Jenkinsfile, a link:/blog/2019/05/09/templating-engine/[Jenkins Templating Engine file], a plugin:pipeline-as-yaml[Pipeline as YAML], or a link:/blog/2019/01/08/mpl-modular-pipeline-library/[Jenkins Modular Pipeline Library].
+
+Yammer::
+Last released **8 years ago**. +
+Suggestion: Use the link:https://developer.yammer.com/docs/rest-api-rate-limits[Yammer REST API] to post messages.
+
+DevStack::
+Last released **9 years ago**.
+
+Ikachan::
+Last released **9 years ago**.
+
+Jenkinspider::
+Last released **6 years ago**.
+
+Perl Smoke Test::
+Last released **7 years ago**.
+
+buddycloud::
+Last released **7 years ago**.
+
+=== Affected plugins, Python runtime
+
+InstallShield::
```

Comment:
is this correct - it uses both python and ruby runtime?  
https://github.com/jenkinsci/jep/pull/356/files#diff-79416440f404d2b50d67e437cd226b4e742c20f94cce5a10f23be10d063d65adR233

Reply:
Nope, only Python. Copied it from the blogpost from @halkeye , will update the JEP text